dev/tests/api-functional/testsuite/Magento/GraphQl/SendFriend/SendFriendTest.php

```diff
@@ -0,0 +1,469 @@
+        input: {
+          product_id: 1
+          sender: {
+            name: "Name"
+            email: "e@mail.com"
+            message: "Lorem Ipsum"
+        }          
+          recipients: [
+              {
+                  name: "Recipient Name 1"
+                  email:"recipient1@mail.com"
+               },
+              {
+                  name: "Recipient Name 2"
+                  email:"recipient2@mail.com"
+              }
+          ]
+        } 
+    ) {
+        sender {
+            name
+            email
+            message
+        }
+        recipients {
+            name
+            email
+        }
+    }
+}
+QUERY;
+
+        $response = $this->graphQlQuery($query);
+        self::assertEquals('Name', $response['sendEmailToFriend']['sender']['name']);
+        self::assertEquals('e@mail.com', $response['sendEmailToFriend']['sender']['email']);
+        self::assertEquals('Lorem Ipsum', $response['sendEmailToFriend']['sender']['message']);
+        self::assertEquals('Recipient Name 1', $response['sendEmailToFriend']['recipients'][0]['name']);
+        self::assertEquals('recipient1@mail.com', $response['sendEmailToFriend']['recipients'][0]['email']);
+        self::assertEquals('Recipient Name 2', $response['sendEmailToFriend']['recipients'][1]['name']);
+        self::assertEquals('recipient2@mail.com', $response['sendEmailToFriend']['recipients'][1]['email']);
+    }
+
+    public function testSendWithoutExistProduct()
+    {
+        $query =
+            <<<QUERY
+mutation {
+    sendEmailToFriend(
+        input: {
+          product_id: 2018
+          sender: {
+            name: "Name"
+            email: "e@mail.com"
+            message: "Lorem Ipsum"
+        }          
+          recipients: [
+              {
+                  name: "Recipient Name 1"
+                  email:"recipient1@mail.com"
+               },
+              {
+                  name: "Recipient Name 2"
+                  email:"recipient2@mail.com"
+              }
+          ]
+        } 
+    ) {
+        sender {
+            name
+            email
+            message
+        }
+        recipients {
+            name
+            email
+        }
+    }
+}
+QUERY;
+        $this->expectException(\Exception::class);
+        $this->expectExceptionMessage('The product that was requested doesn\'t exist. Verify the product and try again.');
+        $this->graphQlQuery($query);
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/SendFriend/_files/product_simple.php
+     */
+    public function testMaxSendEmailToFriend()
+    {
+        /** @var SendFriend $sendFriend */
+        $sendFriend = $this->sendFriendFactory->create();
+
+        $query =
+            <<<QUERY
+mutation {
+    sendEmailToFriend(
+        input: {
+          product_id: 1
+          sender: {
+            name: "Name"
+            email: "e@mail.com"
+            message: "Lorem Ipsum"
+        }
+          recipients: [
```

Comment:
Looks like two elements of recipients will be enough

Reply:
No
It's ok
Check app/code/Magento/SendFriend/etc/config.xml:15